docs/_static/custom.css

```diff
@@ -103,36 +106,67 @@ div.section {
   font-weight: bold;
 }
 
-#search-form {
-  display: flex;
-  flex-direction: row;
-  align-items: baseline;
-  flex-wrap: wrap;
-}
-#search-form select,
-#search-form input[type="text"] {
-  height: 2rem;
-  margin-right: 1rem;
-  margin-bottom: 1rem;
-  background-color: white;
-  border: 1px solid darkgrey;
+
+/* Bootstrap */
+.btn-primary {
+  color: #fff;
+  background-color: #2980b9;
+  border-color: #2980b9;
 }
-#search-form input[type="submit"] {
-  margin-right: 1rem;
+.btn-primary {
+  background-color: #1f86ca;
+  border-color: #2980b9;
 }
-ul.search .title_doc_section::before {
-  content: ' – ';
+
+/* Search */
+ul.search {
+  margin-left: 0;
 }
 p.search-summary {
-  margin-top: 1em;
+  margin: 1em 0 2rem 0;
 }
 #search-results ul {
   list-style-type: none;
   padding-left: 0;
 }
-#search-results ul li {
-  margin-bottom: 1em;
+#search-results ul li h3 {
+  margin: 0.75rem 0 .5rem;
+  font-size: 1.5rem;
 }
+#search-results ul li .breadcrumbs {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+#search-results ul li .breadcrumbs a {
+  font-weight: normal;
+}
+#search-results ul li .breadcrumbs .lastbreadcrumb {
+  white-space: nowrap;
+  display: inline-block;
+  max-width: 12rem;
+  overflow: hidden;              /* "overflow"-Wert darf nicht "visible" sein */
+
+  text-overflow: ellipsis;
+}
+#search-results ul li,
+ul.search li {
+  margin-bottom: 2rem;
+  padding: 2px 0 5px 4rem;
+  background-image: url(file.svg);
+  background-blend-mode: hue;
+  background-color: violetred;
+  background-repeat: no-repeat;
+  background-position: 0px 8px;
+  background-size: 56px;
+}
+ul.search li p.context {
+  margin-left: 0;
+}
+.pathseparator {
+  padding: 0 0.7rem;
+}
+
```

Comment:
I don't think the icon adds anything to the layout. Everything is a file.

If we keep it, then I definitely think it needs to be reduced in size. It dominates and distracts from the information presented in the results.